### PR TITLE
curator operator cluster level roles and role bindings

### PR DESCRIPTION
**Pull request by skanthed**

### CODE REVIEW

File: cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/curator-operator-manager-rb/clusterrolebinding.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: curator-manager-rolebinding
```
Comment (dystewart) [inline cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/curator-operator-manager-rb/clusterrolebinding.yaml:4]: For each role/rolebinding can you include operator in the name?
ie: 
name: curator-operator-manager-rolebinding

**Review by dystewart — CHANGES_REQUESTED** [review]

File: cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/curator-operator-manager-rb/clusterrolebinding.yaml (hunk 1 above)
Comment (skanthed) [reply cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/curator-operator-manager-rb/clusterrolebinding.yaml:4]: okay.

**Comment by skanthed** [comment]
@dystewart Updated all the names.

**Review by dystewart — APPROVED** [review]

File: cluster-scope/base/rbac.authorization.k8s.io/clusterroles/curator-operator-manager-role/clusterrole.yaml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  - jobs/status
+  verbs:
+  - get
+- apiGroups:
+  - ""
+  resources:
+  - persistentvolumeclaims
+  verbs:
+  - create
+  - delete
```
Comment (HumairAK) [inline cluster-scope/base/rbac.authorization.k8s.io/clusterroles/curator-operator-manager-role/clusterrole.yaml:58]: @skanthed why does this operator need write permissions to all pvc's on the cluster?

Comment (skanthed) [reply cluster-scope/base/rbac.authorization.k8s.io/clusterroles/curator-operator-manager-role/clusterrole.yaml:58]: @HumairAK Most of the RBAC are auto-generated when we are building an operator using operator-sdk. We have not written any logic that requires access to other PVCs. Even if we don't have access to other PVCs on the cluster, it should be good.

I can try changing it and test it on the curator cluster. I will update the PR accordingly.

**Pull request merged**